config/csv_file_defaults.yml

```diff
@@ -1,149 +1,154 @@
 generic:
```

Comment:
why was this change made?

Reply:
I changed after running bundle exec rake ci